src/pages/finance/paymentMethod/index.jsx

```diff
@@ -72,196 +70,139 @@ export const PaymentMethodDashboard = () => {
         } catch (e) {
 
         }
-    };
+    }
 
     const handleDeletePaymentMethod = async (id) => {
-        try{
-            await deleteFromApiData(`paymentmethods/${id}`);
+        try {
+            await deleteFromApiData(`paymentmethods/${id}`)
             await getPaymentMethods();
-        }catch(error){
-            console.error(error);
+        } catch (error) {
+            console.error(error)
+        }
+    }
+
+    const renderIconCards = (type) => {
+        switch (true) {
+            case type.toLowerCase().includes('card'):
```

Comment:
#should: it should be nice to think a way to not need to pass those values directly here, just because that approach has a problem if we need to scale up